commands/smurf.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+// import jssha
+const jsSHA = require("jssha")
+
+const {
+  DEFAULT_SCHEMAS,
+  validate
+} = require('./../validation.js')
+
+// dec2hex
+function dec2hex(s) {
+  return (s < 15.5 ? "0" : "") + Math.round(s).toString(16)
+}
+// hex2dec
+function hex2dec(s) {
+  return parseInt(s, 16)
+}
+// leftpad
+function leftpad(str, len, pad) {
+  if (len + 1 >= str.length) {
+    str = Array(len + 1 - str.length).join(pad) + str
+  }
+  return str
+}
+
+function getOtp(key, now = new Date().getTime()) {
+  const epoch = Math.round(now / 1000.0)
+  const time = leftpad(dec2hex(Math.floor(epoch / 30)), 16, "0")
```

Comment:
Magic numbers are bad. Can you assign stuff to consts before using them, or at least document with comments?

Reply:
assigned to const otpResultLength